canonical-websites/www.ubuntu.com
# Updated links to community support

## Done

* Updated links to community support links from https://community.ubuntu.com/ to https://community.ubuntu.com/t/community-support/709
* Deleted unused shared file - _ubuntu_advantage_community.html

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
* [live page copy doc](https://docs.google.com/document/d/1ObUSvkDcWOe01pmhh_Aryqxs87D8X2khbJbn3RgqZE0/edit#heading=h.hgj9tub2jtz2) - [page](http://0.0.0.0:8001/server/livepatch)
* [community support copy doc](https://docs.google.com/document/d/1vrnljXUx71ZZhz3cQ6NupUNw6yNqW2OWkyY8Ozx2bqw/edit#) and [page](http://0.0.0.0:8001/support/community-support)

## Issue / Card

Fixes #2450
